templates/patterns/modal/modal.ui_patterns.yml

```diff
@@ -1,81 +1,68 @@
 modal:
-  label: Modal
-  description: Use Bootstrap’s JavaScript modal plugin to add dialogs to your site for lightboxes, user notifications, or completely custom content. https://v5.getbootstrap.com/docs/5.0/components/modal/
+  label: "Modal"
+  description: "Use Bootstrap’s JavaScript modal plugin to add dialogs to your site for lightboxes, user notifications, or completely custom content. https://v5.getbootstrap.com/docs/5.0/components/modal/"
   settings:
     btn_style:
       type: select
       label: Button style
-      options:
-        primary: primary
-        secondary: secondary
-        success: success
-        danger: danger
-        warning: warning
-        info: info
-        light: light
-        dark: dark
-        link: link
       preview: primary
-    btn_outline:
-      type: boolean
-      label: Button outline
-    btn_size:
-      type: select
-      label: Button size
-      options:
-        sm: medium
-        lg: large
-      preview: lg
-    modal_size:
+    size:
       type: select
       label: Modal size
       options:
-        sm: small
-        lg: large
-        xl: extra large
-    fullscreen:
-      type: select
-      label: Fullscreen size
-      options:
-        always: always
-        sm: below 576px
-        md: below 768px
-        lg: below 992px
-        xl: below 1200px
-    static_backdrop: 
-      type: boolean
-      label: Static backdrop
-      description: When backdrop is set to static, the modal will not close when clicking outside it.
-    vertically_centered: 
-      type: boolean
-      label: Vertically centered
-      description: Vertically center the modal.
-      preview: true
-    scrollable: 
-      type: boolean
-      label: Scrollable
-      description: When modals become too long for the user’s viewport or device, they scroll independent of the page itself.
-      preview: true
+        sm: "sm"
+        lg: "lg"
+        xl: "xl"
+        fullscreen: "fullscreen"
+      preview: lg
+    id:
```

Comment:
id is a reserver work for the pattern, if I set this it breaks, use `modal_id` or something like that